webpages/settings/style.css

```diff
@@ -147,7 +154,7 @@ h1 {
 }
 #confirmImport,
 #applyLanguageSettingsButton {
-  border-color: var(--orange);
+  background: var(--brand-orange);
```

Comment:
This should have the text set to `--white-text` to fix it in light mode

Reply:
I'm still doing some tweaks pls wait

Reply:
> This should have the text set to `--white-text` to fix it in light mode

wdym?

Reply:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/065e259d-1e5b-4cca-ab3e-d02c17159575)

Reply:
> ![image](https://private-user-images.githubusercontent.com/81489795/311536383-065e259d-1e5b-4cca-ab3e-d02c17159575.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwOTY0MzcsIm5iZiI6MTcxMDA5NjEzNywicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MzYzODMtMDY1ZTI1OWQtMWU1Yi00Y2NhLWFiM2UtZDAyYzE3MTU5NTc1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDE4NDIxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhMjZjYTA0YjM0MmYxNDI3MDBmNWUwOTdjZTAxMDBmMzE1ZjhjYzg2ZmUxY2E5MzU1Y2M3MThjYTQxODVjNDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ambzlGnI1967XPkKK7zL4pwrdct5JENHsmaN2Y5awuM)

oh yeah

Reply:
Fixed